crates/hotshot/src/lib.rs

```diff
@@ -320,6 +322,20 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
         };
     }
 
+    /// Emit an external event
+    // A copypasta of `ConsensusApi::send_event`
+    // TODO: remove with https://github.com/EspressoSystems/HotShot/issues/2407
+    async fn send_event(&self, event: Event<TYPES>) {
```

Comment:
Nit: Can we rename this to `send_external_event` just to differentiate it from sending an internal event.